deploy/template.yaml

```diff
@@ -122,6 +123,7 @@ Mappings:
       DNSSUFFIX: "return.build.account.gov.uk"
       OIDCURL: "https://oidc.staging.account.gov.uk/"
       RETURNREDIRECTURL: "https://return.build.account.gov.uk/callback"
+      TESTHARNESSURL: "https://ipvreturn-test-harness-testharness.return.build.account.gov.uk"
```

Comment:
please check this against the test-harness template - it might be just `testharness.return.build.account.gov.uk`

Reply:
URL has been updated to match the test-harness template